### DISCUSSION TITLE
[#995505] Chat screen shows player's arm

### DISCUSSION THREAD

**Discussion by LateLag** [Mojang/minecraft-editor#88] [Bugs]
**Steps to Reproduce:**

1. Check to see if Hide Hand is disabled in the Video settings;
2. Open the chat screen directly from the Tool mode by pressing `;`

**Observed Results:**
The chat screen it makes your arm visible regardless of which mode you were in.

**Expected Results:**
Your arm remains invisible, and only becomes visible when switching to Crosshair mode .

**Comment by minecraftchloe** [comment]
Nice catch - a great reminder for us to check the video settings and what applies or doesn't apply in Tool Mode!

**Comment by minecraftchloe** [comment]
Closing this as "as designed" in consult with the team -- Switching to chat screen automatically changes you to Crosshair mode. If "Hide Hand" option is disabled, the hand should be visible behind chat screen.
